docs/root/intro/arch_overview/observability/access_logging.rst

```diff
@@ -17,29 +17,54 @@ logs<envoy_v3_api_field_config.listener.v3.Listener.access_log>`. The listener a
 HTTP request access logging and can be enabled separately and independently from
 filter access logs.
 
+If access log is enabled, then by default it will be reported to the configured sinks at the end of a TCP
+connection, or HTTP stream. It is possible to extend this behavior and report access logs periodically or at the
+start of a TCP connection or HTTP stream. Reporting access logs right after upstream connection establishment
+does not depend on periodic reporting, and the other way around.
+
+.. _arch_overview_access_log_start:
+
+Start of session access logs
+----------------------------
+
+TCP Proxy
+*********
+
+For TCP Proxy, it is possible to enable an access log record once after a successful upstream connection by using
+:ref:`flush access log on connected <envoy_v3_api_field_extensions.filters.network.tcp_proxy.v3.TcpProxy.flush_access_log_on_connected>`
+
+HTTP Connection Manager
+***********************
+
+For HTTP Connection Manager, it is possible to enable an access log once when a new HTTP request is received, and before iterating the filter chain by using
+:ref:`flush access log on new request <envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.flush_access_log_on_new_request>`
+Note: Some information such as upstream host will not be available yet.
+
+HTTP Router Filter
+******************
+
+For Router Filter, is is possible to enable an upstream access log when a new HTTP request is received, and after successfully establishing a connection
```

Comment:
when a new HTTP request is received -> when a new upstream stream is associated with the downstream stream

Reply:
done